pkg/handlers/generic/lifecycle/csi/nutanix-csi/handler.go

```diff
@@ -149,8 +158,8 @@ func (n *NutanixCSI) handleHelmAddonApply(
 			ClusterSelector: metav1.LabelSelector{
 				MatchLabels: map[string]string{clusterv1.ClusterNameLabel: req.Cluster.Name},
 			},
-			ReleaseNamespace: req.Cluster.Namespace,
```

Comment:
This is the namespace where the CSI driver is deployed to, which by default is `ntnx-cluster`. Its unrelated the Cluster's namespace.